src/configs/starship_root.rs

```diff
@@ -71,6 +71,7 @@ pub const PROMPT_ORDER: &[&str] = &[
     // ↑ Toolchain version modules ↑
     "nix_shell",
     "conda",
+    "spack",
```

Comment:
Not 100% sure about this location in the prompt order, but I can't come up with a better one.

Reply:
Not sure either, I just mimicked the `conda` location as these are interchangeable package managers :man_shrugging:

Reply:
Well, we can always move it around if we need to.

Reply:
As a user of both conda and spack, I don't know if I really have a preference.